### PR TITLE
Update the words

### PR DESCRIPTION
**Pull request by clyang82**
Signed-off-by: Chun Lin Yang <clyang@cn.ibm.com>

### CODE REVIEW

File: go/memcached-operator/Makefile [hunk 1]
```diff
@@ -4,18 +4,18 @@ NAMESPACE=memcached
...
 	- kubectl apply -f deploy/crds/cache.example.com_memcacheds_crd.yaml -n ${NAMESPACE}
 	@echo ....... Applying Rules and Service Account .......
 	- kubectl apply -f deploy/role.yaml -n ${NAMESPACE}
 	- kubectl apply -f deploy/role_binding.yaml  -n ${NAMESPACE}
 	- kubectl apply -f deploy/service_account.yaml  -n ${NAMESPACE}
 	@echo ....... Applying Operator .......
 	- kubectl apply -f deploy/operator.yaml -n ${NAMESPACE}
-	@echo ....... Creating the Database .......
+	@echo ....... Creating the CRs .......
 	- kubectl apply -f deploy/crds/cache.example.com_v1alpha1_memcached_cr.yaml -n ${NAMESPACE}
```
Comment (camilamacedo86) [inline go/memcached-operator/Makefile]: Thank you for your contribution. 🥇 
Could you update the uninstall a well?

Comment (clyang82) [reply go/memcached-operator/Makefile]: The uninstall part is correct. could you point me where need to update? Thanks.

Comment (camilamacedo86) [reply go/memcached-operator/Makefile]: Sorry, it is true. I assume that the text was equally wrong :-)

**Comment by camilamacedo86** [comment]
I will merge this one with just on approval because it is very small. Just typo issue in the Makefile.

**Pull request merged**